projects/storefrontapp/project.json

```diff
@@ -7,21 +7,29 @@
   "generators": {},
   "targets": {
     "build": {
-      "executor": "@angular-builders/custom-webpack:browser",
+      "executor": "@angular-builders/custom-esbuild:application",
+      "logLevel": "error",
```

Comment:
1. Do we really have to change the default logLevel?
2. And what was the previous default logLevel?